cap/modules/deposit/cli.py

```diff
@@ -142,6 +146,44 @@ def create_deposit(file, ana, roles, users, owner, save_errors):
     click.secho(f"\nProcess finished and record(s) added.", fg='green')
 
 
+@fixtures.command('token-generator')
```

Comment:
Example case: `docker-compose -f docker-compose.dev.yml run web-api cap fixtures token-generator -u info45@inveniosoftware.org --token-name test123`

Reply:
move to `users` or `oauth2server` module